### PR TITLE
feat(tabs): support action-items slot (VIV-2031)

### PR DESCRIPTION
**Pull request by RichardHelm**
This PR also fixes a bug with the active indicator when tabs are added, and it's general glitchiness

### CODE REVIEW

File: libs/components/src/lib/tab/tab.scss [hunk 1]
```diff
@@ -102,33 +102,6 @@ slot[name='icon'] {
 	}
 }
 
-:host(:not(.vertical)) {
```
Comment (RichardHelm) [inline libs/components/src/lib/tab/tab.scss:105]: Remove active-indicator from tab and instead keep showing the one from tabs.
Not sure why it was ever added here

File: libs/components/src/lib/tabs/tabs.ts [hunk 2]
```diff
@@ -166,14 +165,68 @@ export class Tabs extends FoundationTabs {
 
 		this.#tabListWrapper.scrollTo({ top, left, behavior: 'smooth' });
 	}
-	// adapted FAST fix https://github.com/microsoft/fast/pull/6606
```
Comment (RichardHelm) [inline libs/components/src/lib/tabs/tabs.ts:169]: This fix has been adopted in our version of FAST so this is no longer needed

File: libs/components/src/lib/tab/tab.scss (hunk 1 above)
Comment (rachelbt) [reply libs/components/src/lib/tab/tab.scss:105]: If I remember correctly it was part of the indicator animation. Maybe the change in Fast made this not needed anymore ⚡

**Review by rachelbt — COMMENTED** [review]
👍

**Review by rachelbt — COMMENTED** [review]
the code is all OK, but the ui png are not presenting the slotted button - maybe you need to add "button" to the ui tests import

**Comment by RichardHelm** [comment]
@rachelbt You're right, I've updated it

**Review by TaylorJ76 — APPROVED** [review]
Nice!

**Pull request merged**